protocol/synthetix/contracts/modules/core/RewardsManagerModule.sol

```diff
@@ -150,6 +150,29 @@ contract RewardsManagerModule is IRewardsManagerModule {
         return _getRewardRate(poolId, collateralType, distributor);
     }
 
+    /**
+     * @inheritdoc IRewardsManagerModule
+     */
+    function getAvailableRewards(
+        uint128 accountId,
+        uint128 poolId,
+        address collateralType,
+        address distributor
+    ) external view returns (uint256) {
+        Vault.Data storage vault = Pool.load(poolId).vaults[collateralType];
+        bytes32 rewardId = _getRewardId(poolId, collateralType, distributor);
+
+        if (address(vault.rewards[rewardId].distributor) != distributor) {
+            revert ParameterError.InvalidParameter("invalid-params", "reward is not found");
+        }
+
+        uint256 rewardAmount = 0;
+
+        rewardAmount += vault.getReward(accountId, rewardId);
```

Comment:
I think this can be simplified to:
`uint256 rewardAmount = vault.getReward(accountId, rewardId);`
Or even
`return vault.getReward(accountId, rewardId);`